packages/inter-protocol/src/auction/auctioneer.js

```diff
@@ -116,47 +116,47 @@ const distributeProportionalShares = (
 
 /**
  * Return a set of transfers for atomicRearrange() that distribute
- * collateralReturn and currencyRaised proportionally to each seat's deposited
+ * collateralReturn and bidRaised proportionally to each seat's deposited
  * amount. Any uneven split should be allocated to the reserve.
  *
  * This function is exported for testability, and is not expected to be used
  * outside the contract below.
  *
  * Some or all of the depositors may have specified a goal amount.
- *  A if none did, return collateral and currency prorated to deposits.
- *  B if currencyRaised < proceedsGoal everyone gets prorated amounts of both.
- *  C if currencyRaised matches proceedsGoal, everyone gets the currency they
+ *  A if none did, return collateral and bid prorated to deposits.
+ *  B if bidRaised < proceedsGoal everyone gets prorated amounts of both.
+ *  C if bidRaised matches proceedsGoal, everyone gets the bid they
```

Comment:
it seems somewhat remarkable that this change didn't suggest/require any changes to the surrounding prose. The new term just drops right in.